docs/app/pages/css/css-sections/icons/ux-icons/icon-preview/icon-preview.component.html

```diff
@@ -0,0 +1,42 @@
+<button uxFocusIndicator
+        tabindex="0"
+        class="icon-demo btn btn-link button-secondary btn-text"
+        [uxPopover]="iconCodeSnippet"
+        popoverClass="icon-code-snippets"
+        placement="bottom">
+    <i class="{{ iconClass }} p-r-xs" [ngClass]="classname"></i>
+    <span>{{ iconset === 'ux-icons' ? name : classname }}</span>
+</button>
+
+<ng-template #iconCodeSnippet>
+    <table class="icon-code-snippets-table">
+        <tbody>
+        <tr *ngIf="iconset === 'ux-icons'">
+            <td class="p-l-xs">&lt;ux-icon name="{{name}}"&gt;&lt;/ux-icon&gt;&nbsp;&nbsp;</td>
+            <td><i class="ux-icon ux-icon-clipboard m-r-xs"
+                   [uxTooltip]="tooltipTemplate"
+                    (click)="copy('ux-icon-component')"></i></td>
+        </tr>
+        <tr *ngIf="iconset === 'ux-icons'">
+            <td class="p-l-xs">&lt;i class="ux-icon {{classname}}"&gt;&lt;/i&gt;&nbsp;&nbsp;</td>
```

Comment:
I think the code snippets in the popover would look better if we used the code highlighting like we do elsewhere

Reply:
Used the code highlighting for popover